components/search/list-renderer/human-genomic-variant.js

```diff
@@ -0,0 +1,44 @@
+import {
+  SearchListItemContent,
+  SearchListItemMain,
+  SearchListItemMeta,
+  SearchListItemStatus,
+  SearchListItemTitle,
+  SearchListItemType,
+  SearchListItemUniqueId,
+} from "./search-list-item";
+import { PropTypes } from "prop-types";
+
+export function humanGenomicVariantTitle(variant) {
+  const first = variant.chromosome
+    ? variant.chromosome
+    : variant.refseq_id || variant.reference_sequence;
+  const titleSeq = [first, variant.position, variant.ref, variant.alt];
+  return titleSeq.join(":");
+}
+
+export default function HumanGenomicVariant({ item: variant }) {
+  return (
+    <SearchListItemContent>
+      <SearchListItemMain>
+        <SearchListItemUniqueId>
+          <SearchListItemType item={variant} />
+          {variant.uuid}
+        </SearchListItemUniqueId>
+        <SearchListItemTitle>
+          {humanGenomicVariantTitle(variant)}
+        </SearchListItemTitle>
+        {variant.rsid && (
+          <SearchListItemMeta>
+            <div key={variant.rsid}>{variant.rsid}</div>
```

Comment:
You don’t need a key here since `SearchListItemMeta` is a simple wrapper — not a hidden loop. But also I think all you need is `{variant.rsid}` without the `div` to achieve the same thing.

Reply:
I initially tried it w/out a div, and it seemed like it wanted  it to be a div or html element, but I could be mistaken

Reply:
Oh! I was totally wrong. I forgot `SearchListItemMeta` _does_ have a hidden loop, so yeah, you need the `div`. You don’t need the key because it only has one item.